sklearn/preprocessing/_function_transformer.py

```diff
@@ -394,6 +394,7 @@ def __sklearn_tags__(self):
         tags = super().__sklearn_tags__()
         tags.no_validation = not self.validate
         tags.requires_fit = False
+        tags.input_tags.sparse = True
```

Comment:
This should be a bit more tricky because it will depend on `self.validate` and `self.accept_sparse`


```suggestion
        tags.input_tags.sparse = not self.validate or (self.validate and self.accept_sparse)
```